app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CategoriesController < ApplicationController
+    before_action :set_category, only: [:show, :destroy]
+
+    def index
+      @categories = Category.all
+    end
+
+    def show
+      @books = @category.books
+    end
+
+    def destroy
+      @category.destroy
```

Comment:
do you know if destroy returns a boolean or nil?

Reply:
It returns the deleted activerecord
![image](https://user-images.githubusercontent.com/29805061/87211632-b2875100-c31a-11ea-91b3-15a895b5b992.png)